### ISSUE TITLE
Add feature to underline/highlight search terms in resul list

### ISSUE THREAD

**Issue by tgice** [sagalbot/vue-select#576]
Hi, I don't suppose I've missed a feature that a lot of the typeahead libraries have where as the user is typing, the search terms are underlined or highlighted in the resulting options display?

Similar to how harvesthq/chosen works:

https://harvesthq.github.io/chosen/

If not, would the author or others consider adding this feature?

**Comment by tgice** [comment]
Eh, as may not surprise most of you, I figured out a fairly simple way to do this just using Vue features.

I used something like this inside my <template slot="option.. tags:

`<span v-html="emphasize(option.Number, searchStr)"></span>`

where I was saving searchStr in my @@search-referenced method for later reference in this other formatting method, emphasize, which I defined like this:

```
    emphasize(val, searchStr) {
        return val.replace(searchStr, "<u>" + searchStr + "</u>");
    }
```

so pretty simple all around, and by using the v-html property above, I get the HTML coming through unescaped so I get my underline.

Does anyone know of a better way of referencing the current user-inputted search string in the v-select without having to do that manual saving of it in the Vue instance?
